### ISSUE TITLE
Docs: encoded urls doesn't open doc

### ISSUE THREAD

**Issue by chrisands** [microsoft/TypeScript-Website#1876]
**Page URL:**

https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#optional-chaining
https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html%23optional-chaining

**Issue:** 

In pdf links are being encoded (like second link) and when I try to open encoded url it says `The resource you are looking for has been removed, had its name changed, or is temporarily unavailable.`.  It should work on such links.

**Recommended Fix:** `

Parse encoded symbols?

**Comment by orta** [comment]
Great spot - I guess we'll need to do some pre-parsing somewhere in the epub/pdf generator to undo some encoding

**Comment by chrisands** [comment]
In [typescript-handbook](https://www.typescriptlang.org/assets/typescript-handbook.pdf) everything seems ok. 
I meant that I created a presentation for my work and put a link to the handbook. And after exporting it to pdf to share with my colleges, URL was encoded and colleges couldn't open links. I couldn't find any information on how to disable URL encoding in different apps. I tried export presentations from Canva and Keynote. 
I thought maybe it possible to fix on somewhere at balancer to allow the use of encoded symbols.
